utils/deploys/deployExtensions.ts

```diff
@@ -264,6 +266,46 @@ export default class DeployExtensions {
     );
   }
 
+  public async deployFlashMintNotional(
+    wethAddress: Address,
```

Comment:
We should probably start making this arguments an object rather than a list of arguments. As it becomes much easier to make a mistake passing in the wrong address into the wrong slot here

Reply:
Yes we should. I tried to have a single `addresses` struct argument in the contstructor of type `DEXAdapter.Addresses` but I got some very cryptic compiler failure (see below).

I suspect this is a result from us using such an outdated solidity version
<img width="874" alt="image" src="https://user-images.githubusercontent.com/15629702/197123671-49cc726e-b4a2-4bbb-8d96-787f1f9e7288.png">